examples/src/main/python/gpu/nubma_example.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np 
```

Comment:
I really like this example. Can we do 2 things
1. Have a java/scala equivalent?
2. Have a jupyter notebook version as well? 

Probably also worth adding some code to try to detect if the GPU is present though before running this.

Reply:
[This should work for detecting GPUs with Numba](http://numba.pydata.org/numba-doc/0.19.1/cuda-reference/host.html#numba.cuda.is_available). It seems like most other gpu-enabled tools have similar functions so when I add java and scala examples, I'll make sure to include them.